src/request/request-tab.js

```diff
@@ -34,7 +34,8 @@ module.exports = {
 // TODO: Need to come up with a better self registration process
 require('./components/request-detail-panel-execution');
 var request = require('./components/request-detail-panel-request');
-require('./components/request-detail-panel-logging');
+var LoggingComponent = require('./components/request-detail-panel-logging').Logging;
+var LoggingViewModel = require('./view-models/LoggingViewModel').LoggingViewModel;
```

Comment:
I've always preferred to treat views and view-models as peers rather than imply a hierarchy or "owner".  They are used outside of the view--for testing!  The views should care only that what's given to them meets the agreed-upon contract for property binding, not what or how or by whom those contracts are implemented.

Obviously the file hierarchy doesn't necessarily have any bearing on that, but I find that keeping them as peers tends to prevent the thinking that one serves the other, which might then skew the design in that, undesired, direction. 

At the end of the day, though, it's not that big of a deal.